README.md

```diff
@@ -62,6 +65,7 @@ analysis-runner \
 --description "Joint calling, full" \
 --access-level full \
 joint-calling/driver_for_analysis_runner.sh workflows/batch_workflow.py\
+--scatter-count 200 \
```

Comment:
This might actually be a bit much at least initially, right?

Reply:
Perhaps... How many would you use for our 4 batches?

Reply:
Changed to 100